scripts/docker-as-drupal

```diff
@@ -203,6 +212,19 @@ function setup() {
     echo '$databases["default"]["default"] = \Drupal\Core\Database\Database::convertDbUrlToConnectionInfo(getenv("DATABASE_URL"), DRUPAL_ROOT);' >> ./web/sites/default/settings.php
   fi
 
+  if ! grep -Eq "\\\$settings\['trusted_host_patterns'\] = '.+'" ./web/sites/default/settings.php; then
```

Comment:
```suggestion
  if [ ! -z "$SITE_TRUSTED_HOST_PATTERNS" ] && [ ! grep -Eq "\\\$settings\['trusted_host_patterns'\] = '.+'" ./web/sites/default/settings.php ]; then```

@yannlugrin can we do that ? Otherwise it break Watchdreamer (as we don't fix patterns) but it will but 127.0.0.1 and localhost by default - and so *mwatchdreamer.test are not allowed.